nextcord/client.py

```diff
@@ -578,13 +578,15 @@ async def on_application_command_error(
 
         This only fires if you do not specify any listeners for command error.
         """
-        if interaction.application_command and interaction.application_command.has_error_handler():
+        if interaction.application_command is None:
+            return  # Not supposed to ever happen
+
+        if interaction.application_command.has_error_handler():
             return
```

Comment:
Don't understand the question

Reply:
These if statements used to be combined together with the `and` operator. Why did you change that?

Reply:
I don't remember anymore, but I think it was because I was fiddling around with it, changed it a lot, and it just ended up like this without me giving it much more thought.

Or I did for the sake of readability, not sure after all this time